src/test/java/xml/XmlWriterTest.java

```diff
@@ -33,10 +33,11 @@ public void writeProject() throws Exception {
         project.setDirectorTimeline(dt);
 
```

Comment:
Shouldn't you add some cameraTimelines and cameras here to really test the added functionalities?